backend/src/routes/task-router.ts

```diff
@@ -35,7 +35,15 @@ export const TaskRoute = (app: Express, controller: TaskController) => {
      *         example: Boil water
      *        recipe:
      *         type: string
-     *         example: recipe01
+     *         example: {
```

Comment:
Did you change this because the old line did not work? I think I broke it, Tong will fix it.

Reply:
yeah lol. Do you want to change it back to how it was before?